js/jquery.dvfCharts.js

```diff
@@ -370,14 +411,118 @@
       return this;
     },
 
+    /**
+     * Parses the column override settings and applies them accordingly.
+     *
+     * @returns {Plugin}
+     */
+    parseColumnOverrideOptions: function () {
+      var column_overrides = this.options.chart.data.column_overrides;
```

Comment:
is there ever a case where this is undefined?

Reply:
Hmm yes I think so. I don't think there would be any errors since the `$.each(...` would simply iterate over nothing, but for future maintainability probably best to explicitly check. Please see update.